## disabledby-color_sample_pack.diff by Stefano Zacchiroli <zack@debian.org>

Index: vim-scripts/plugin/color_sample_pack.vim
===================================================================
--- vim-scripts.orig/plugin/color_sample_pack.vim
+++ vim-scripts/plugin/color_sample_pack.vim
@@ -1,6 +1,11 @@
 " Maintainer: Robert Melton ( iam -at- robertmelton -dot- com)
 " Last Change: 2010 Jan 20th
 
+if exists("loaded_color_sample_pack")
+  finish
+endif
+let loaded_color_sample_pack = 1
+
 " default schemes
 amenu T&hemes.D&efault.Blue :colo blue<CR>
 amenu T&hemes.D&efault.DarkBlue :colo darkblue<CR>
